packages/rest/src/__tests__/acceptance/request-parsing/request-parsing.acceptance.ts

```diff
@@ -64,7 +64,7 @@ describe('request parsing', () => {
   });
 
   it('allows built-in body parsers to be removed', async () => {
-    app.unbind(RestBindings.REQUEST_BODY_PARSER_JSON);
+    app.restServer.unbind(RestBindings.REQUEST_BODY_PARSER_JSON);
```

Comment:
This looks like a breaking change to me. Can we find a way how to keep body parsers bound at application level?